Decompiler/PscDecompiler.cpp

```diff
@@ -1287,13 +1294,6 @@ void Decompiler::PscDecompiler::cleanUpTree(Node::BasePtr program)
  */
 void Decompiler::PscDecompiler::generateCode(Node::BasePtr program)
 {
-#ifdef TRACE_DECOMPILATION
```

Comment:
The removed parts of the trace logging should probably be added back in future commits, such as what was removed here.